databases/docker-pgbouncer/prisma/schema.prisma

```diff
@@ -1,6 +1,6 @@
 datasource db {
   provider = "postgresql"
-  url      = "postgresql://postgres:postgres@127.0.0.1:6433/blog?schema=public"
+  url      = "postgresql://this-will-not-be-used-but-overwritten-in-the-prisma-client-constructor"
```

Comment:
Somewhat confusing that this is defined in the schema but not being used by Prisma client.

Reply:
Suggestion on how to improve this? This makes it more explicit that the two Clients have their own connection string in my opinion.